lib/passport-configurator.js

```diff
@@ -405,6 +406,26 @@ PassportConfigurator.prototype.configureProvider = function(name, options) {
         }
       );
       break;
+    case 'fb-token':
```

Comment:
No, because this was the whole reason it didn't work. The switch was going for the default, calling `passport-oauth2` instead of `passport-facebook-token`. This is why any middleware after the `passport.authenticate` call wouldn't get called.  
The oauth2 strategy would not finish since it would redirect the browser to facebook, however, that never needed to be done since we already had the FB Access token.  
The best solution I've found was to completely override the component in this case, making the strategy selection myself.